JBDatePicker/Classes/JBDatePickerWeekDaysView.swift

```diff
@@ -135,7 +135,7 @@ public final class JBDatePickerWeekDaysView: UIStackView {
         for (index, label) in weekdayLabels.enumerated() {
             
             let labelText = weekdayNameSymbols[index]
-            label.attributedText =  NSMutableAttributedString(string: labelText, attributes:[NSAttributedStringKey.font:fontToUse])
+            label.attributedText =  NSMutableAttributedString(string: labelText, attributes:[NSAttributedString.Key.font:fontToUse])
```

Comment:
Non-blocking, is this line showing semi-colon warning?

Reply:
I'm guessing swiftlint didn't run on this, nor would it run on our dependencies when we bring it in. Good eye though. This project now has no warnings.